Cider tests fail

When following the instructions in the README, ie:

```
$ cd /path/to/cider
$ cask
$ make test
```

The tests fail for me with this output:

```
Compute dependencies
cask exec t --no-site-file --no-site-lisp --batch \
-L . \
-f batch-byte-compile cider-util.el
cask exec: error: [Errno 2] No such file or directory
Makefile:49: recipe for target 'cider-util.elc' failed
make: *** [cider-util.elc] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -46,6 +46,4 @@ elpaclean : clean
 	rm -rf .cask # Clean packages installed for development
 
 %.elc : %.el
-	$(CASK) exec $(EMACS) --no-site-file --no-site-lisp --batch \
-	$(EMACSFLAGS) \
-	-f batch-byte-compile $<
+	$(CASK) build

--- a/README.md
+++ b/README.md
@@ -967,6 +967,9 @@ Run all tests with:
 $ make test
 ```
 
+(Note: tests may not run correctly inside Emacs' shell-mode buffers. Running
+in a terminal is recommended.)
+
 ## License
 
 Copyright © 2012-2014 Tim King, Phil Hagelberg, Bozhidar Batsov, Hugo

